src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -102,10 +100,15 @@ class Generator(object):
 def _select_generator(strategy_pool, fuzzer_path):
   """Pick a generator to generate new testcases before fuzzing or return
   Generator.NONE if no generator selected."""
+  if environment.platform() == 'FUCHSIA':
+    # Unsupported.
+    return Generator.NONE
+
   # We can't use radamsa binary on Windows. Disable ML for now until we know it
   # works on Win.
   # These generators don't produce testcases that LPM fuzzers can use.
-  if IS_WIN or engine_common.is_lpm_fuzz_target(fuzzer_path):
+  if (environment.platform() == 'WINDOWS' or
+      engine_common.is_lpm_fuzz_target(fuzzer_path)):
```

Comment:
Thanks I probably would have made this another if condition if there's another similar one above.